### ISSUE TITLE
Add acceptance of Not recognized upload of media files dialog message.

### ISSUE THREAD

**Issue by kkrawczyk123** [opendatakit/central-frontend#176]
Currently, there is no way to accept dialog informing about not recognized upload of media files. It is still displayed even when changing to Overview, Settings and Submissions tab. Some “OK” or “X” button could be added. Screenshot attached:
![screenshot from 2018-11-09 13-07-46](https://user-images.githubusercontent.com/33342488/48402470-fd0e8f80-e72b-11e8-8982-41606abae699.png)

**Comment by matthew-white** [comment]
When working on this, we should make sure to address both aspects of this issue:

- > Some “OK” or “X” button could be added.
- > It is still displayed even when changing to Overview, Settings and Submissions tab.
  - The popup should be automatically hidden when the user changes tabs, even if the Okay button is not clicked.

@clint-tseng, I just wanted to check that the addition of the Okay button sounds good to you.

**Comment by matthew-white** [comment]
We've made progress on this issue!

> Some “OK” or “X” button could be added.

Now that #194 is merged, there is an Okay button.

> It is still displayed even when changing to Overview, Settings and Submissions tab.

If, for example, you drop multiple files onto the table, the popup will appear. If you then click the Overview tab, the popup disappears. However, if you then click the "Media Files" tab and return there, the popup reappears. I think that is unexpected behavior, and I've created #193 about this behavior and related behavior in Frontend.
